travis_ci_bootstrap_edm.sh

```diff
@@ -0,0 +1,15 @@
+if [[ ${TRAVIS_OS_NAME} == "osx" ]]
```

Comment:
It would be good to add a shebang here, and put a `set -e` on the first line to bail out on errors.